crates/db_views_actor/src/person_view.rs

```diff
@@ -22,6 +22,15 @@ impl PersonViewSafe {
     Ok(Self { person, counts })
   }
 
+  pub fn read_by_name(conn: &PgConnection, person_name: String) -> Result<Self, Error> {
+    let (person, counts) = person::table
+      .filter(person::name.eq(person_name))
+      .inner_join(person_aggregates::table)
+      .select((Person::safe_columns_tuple(), person_aggregates::all_columns))
+      .first::<PersonViewSafeTuple>(conn)?;
+    Ok(Self { person, counts })
+  }
```

Comment:
The filter should be by `PersonId`, so this isn't necessary.

 Look at the Search Perform impl, this should look very similar to that.